README.md

```diff
@@ -246,6 +246,15 @@ allowedClasses: {
 }
 ```
 
+Furthermore, regex expressions are supported via strings that start with `^` and end with `$`:
+```js
+allowedClasses: {
+  p: [ '^regex\\d{2}$' ]
+}
+```
+
+The above example whitelist all classes that matches this regex: `RegExp('^regex\\d{2}$')`.
```

Comment:
Minor issue: please change this term to "permits" as we have removed the term "whitelist" elsewhere in the documentation.

Reply:
Updated README